src/plugins/doaction/demo/file-4.json

```diff
@@ -0,0 +1,3 @@
+{
+  "data": "&From JSON file 4"
+}
```

Comment:
Needs French equivalent (file-4-fr.json)

`{
  "data": "&Du fichier JSON 4"
}`

... and is it `*` or `&`?

Reply:
Use & to be constant. But that was only to differentiate the data inside those multiple files